ArcGISRuntimeSDKQt_CppSamples/Scenes/ChooseCameraController/ChooseCameraController.json

```diff
@@ -0,0 +1,8 @@
+{
+    "source" :  "ChooseCameraController.qml",
+    "otherSourceFiles" :  [
```

Comment:
Note this JSON file is required to inform the sample viewer not just what files to display, but also what data to download!

Take a look at the Animate3D sample's json to see the missing bit!